mslib/msui/mpl_qtwidget.py

```diff
@@ -1030,6 +1030,12 @@ def redraw_xaxis(self):
                                                   lons[::tick_index_step])],
                                     rotation=25, horizontalalignment="right")
 
+            # Remove all vertical lines
```

Comment:
Who is drawing these lines? We should store the lines "properly" in a member to remove it cleanly...?

Reply:
Done, also for Side View.